File: projectiv-c/projectiv-c/BananasViewController.m
#import "BananasViewController.h"

@interface BananasViewController ()

@property (weak, nonatomic) IBOutlet UILabel *fruitAmount;
@property (weak, nonatomic) IBOutlet UITextView *bananasDlTextView;

@end

@implementation BananasViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.

    if (self.deepLinkData != nil) {
        self.bananasDlTextView.attributedText = [self attributionDataToString:self.deepLinkData];
        self.bananasDlTextView.textColor = UIColor.labelColor;
        self.fruitAmount.text = [self getFruitAmount:self.deepLinkData];
    }
}

//- (IBAction)copyShareInviteLink:(UIButton *)sender {
//    [super copyShareInviteLinkWithFruitName:@"apples"];
//}

@end

File: swift/basic_app/basic_app/BananasViewController.swift
//
//  BananasViewController.swift
//  basic_app
//
//  Created by Liaz Kamper on 01/06/2020.
//  Copyright © 2020 OneLink. All rights reserved.
//

import UIKit

class BananasViewController: DLViewController {
    
    @IBOutlet weak var fruitAmount: UILabel!
    @IBOutlet weak var bananasDlTextView: UITextView!
    
    override func viewDidLoad() {
        super.viewDidLoad()

        if (deepLinkData != nil) {
            bananasDlTextView.attributedText = attributionDataToString(data: deepLinkData!)
            bananasDlTextView.textColor = .label
            fruitAmount.text = getFruitAmount(data: deepLinkData!)
        }
    }

    @IBAction func copyShareInviteLink(_ sender: UIButton) {
        super.copyShareInviteLink(fruitName: "bananas")
    }
    
}
